src/endpoint/devices.ts

```diff
@@ -93,49 +178,82 @@ export interface HealthState {
 }
 
 export interface Device {
-	deviceId?: string
+	deviceId: string
+	presentationId: string
+	manufacturerName: string
+	type: DeviceIntegrationType
+	restrictionTier: number // integer
 	name?: string
 	label?: string
-	manufacturerName?: string
-	presentationId?: string
 	deviceManufacturerCode?: string
 	locationId?: string // <^(?:([0-9a-fA-F]{32})|([0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}))$>
+	eventId?: string // <^(?:([0-9a-fA-F]{32})|([0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}))$>
+	ownerId?: string
 	roomId?: string // <^(?:([0-9a-fA-F]{32})|([0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}))$>
+	/**
+	 * @deprecated please look under dth
```

Comment:
I suggest being explicit, something like `use dth.deviceTypeId instead`

Reply:
Sorry, I just copied the text from the API docs...I've updated this to be more specific.